content/influxdb/cloud/get-started.md

```diff
@@ -13,10 +13,11 @@ influxdb/cloud/tags: [get-started, install]
 
 After you've [signed up for InfluxDB Cloud](/influxdb/cloud/sign-up/), you're ready to get started:
 
-1. Do one of the following:
+- Do one of the following:
+   - [Write and query data using the programming language of your choice](#write-and-query-data-using-the-programming-language-of-your-choice).
    - Add [sample data](#add-sample-data).
    - [Use your own data](/influxdb/cloud/write-data/) to explore InfluxDB Cloud.
-2. [Create a notebook](#create-a-notebook):
```

Comment:
Creating a notebook isn't part of the onboarding flow.

Reply:
Hi @Luckette, this is our general getting started doc. The notebooks section wasn't added or changed as part of this update. Like the sections about community templates and writing tasks, it's simply an option we're introducing the reader to for starting to interact with their data. 

You can see the full live current doc in context here: https://docs.influxdata.com/influxdb/cloud/get-started/

Let me know if you have any questions!

Reply:
Oh ok got it, thanks for explaining!